docker/test/fasttest/run.sh

```diff
@@ -186,6 +186,8 @@ function clone_submodules
             contrib/dragonbox
             contrib/fast_float
             contrib/NuRaft
+            contrib/thrift
+	    contrib/hive-metastore
```

Comment:
Actually should better remove this. Broken fast test meant that the code is not properly covered with `if (USE_HIVE)`

Reply:
Done.